optimum/intel/neural_compressor/quantization.py

```diff
@@ -309,10 +310,14 @@ def quantize(
             self._quantized_model = convert_to_quantized_model(
                 self._original_model, quantization_config, device=quantization_config.device
             )
-
+            quantization_config.remove_redundant_parameters()
+            self._quantized_model.config.quantization_config = quantization_config
             self._quantized_model.quantization_config = quantization_config
             self._quantized_model.save_pretrained = types.MethodType(save_low_bit, self._quantized_model)
             self._quantized_model.save_pretrained(save_directory)
+            # self._quantized_model saved to "output_dir" with GPTQ format
+            # which is replaced by inplace so loading it to restore optimized model
+            self._quantized_model = INCModelForCausalLM.from_pretrained(save_directory)
```

Comment:
not sure this is needed as we won't use it later on, to fix the `run_clm` example why not load it there instead ?